fastparquet/test/test_api.py

```diff
@@ -388,11 +388,13 @@ def test_bad_file_paths(tempdir):
 
 def test_compression(tempdir):
     pytest.importorskip('lz4')
+    pytest.importorskip('zstandard')
```

Comment:
The test will not run now if either lz4 OR zstandard are missing. I think this is OK in this context, but perhaps there should be at least a comment here.